tests/science/science.cfg

```diff
@@ -1 +1,21 @@
-[test]
+[science_test_snotel]
+driver = classic
+test_description = snotel point simulations
+global_parameter_file = global_param.classic.5.0_snotel.txt
+expected_retval = 0
+check = not_implemented
```

Comment:
The no-nans check is still failing, I'll address that in a follow up PR if that's cool with you.